src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_pending.properties

```diff
@@ -0,0 +1,38 @@
+pendingScripts=Scripts pending approval
+
+# pending scripts
+## table headers
+pendingHash=Hash
+pendingHash_tooltip=The hash could be seen as a signature of the script. It's used as an identifier to reference the content of the script.
+pendingExpandCode=Code
+pendingLanguage=Language
+pendingScriptLength=Length
+pendingScriptLength_tooltip=Length of the script in characters
+contextUser=Requester
+contextItem=Context
+requestApprovalTime=Request date
+requestApprovalTime_tooltip=Inform about when the approval request was done.
+
+## table content
+noRequesterSinceMetadata=N/A
+noRequesterSinceMetadata_tooltip=The requester was not recorded. It could be due to the request being created before the metadata introduction or the code calling this script does not provide the user information.
```

Comment:
```suggestion
noRequesterSinceMetadata_tooltip=The requester was not recorded. It could be due to the request being created before the metadata introduction or the code calling this script did not provide user information.
```